src/TBuffer.cpp

```diff
@@ -2521,7 +2524,6 @@ TBuffer TBuffer::copy(QPoint& P1, QPoint& P2)
         if (!linkId) {
             id = 0;
         }
-        // This is rather inefficient as s is only ever one QChar long
```

Comment:
This hint has not been solved merely erased, yes?

Reply:
Yes, that said if we don't have actual performance concerns with copy() right now, so the efficiency is not a practical issue - more of a theoretical one that doesn't manifest in reality